libs/vkd3d/vkd3d_private.h

```diff
@@ -1876,6 +1876,7 @@ struct vkd3d_rendering_info
     VkRenderingAttachmentInfoKHR dsv;
     VkRenderingFragmentShadingRateAttachmentInfoKHR vrs;
     uint32_t state_flags;
+    uint32_t rtv_mask;
```

Comment:
I think we should be able to not consider RTV null fallbacks at all. Binding a NULL attachment in vkCmdBeginRenderingKHR is legal I think. The only fallback we need to keep around is the "NULL dsv + depth enable" stuff, but other than that, we should be okay just binding NULL attachments.

Reply:
> If the imageView member of any element of pColorAttachments is [VK_NULL_HANDLE](https://www.khronos.org/registry/vulkan/specs/1.2-extensions/html/vkspec.html#VK_NULL_HANDLE), writes to the corresponding location by a fragment are discarded.

Reply:
For depth-stencil, we can also bind NULL:

> Values in imageView are loaded and stored according to the values of loadOp and storeOp, within the render area for each device specified in [VkRenderingInfo](https://www.khronos.org/registry/vulkan/specs/1.2-extensions/html/vkspec.html#VkRenderingInfo). If imageView is [VK_NULL_HANDLE](https://www.khronos.org/registry/vulkan/specs/1.2-extensions/html/vkspec.html#VK_NULL_HANDLE), other members of this structure are ignored; writes to this attachment will be discarded, and no load, store, or resolve operations will be performed.

Reply:
I'm aware of this, but it's such a complex rework that I'd like to revisit this at some other point.

Reply:
k, we can do this afterwards.